pkg/verifier/cosign/cosign.go

```diff
@@ -521,27 +550,27 @@ func decodeASN1Signature(sig []byte) ([]byte, error) {
 }
 
 // getKeysMapsDefault returns the map of keys and cosign options for the reference
-func getKeysMapsDefault(ctx context.Context, trustPolicies *TrustPolicies, reference string, namespace string) (map[PKKey]keymanagementprovider.PublicKey, cosign.CheckOpts, error) {
+func getKeysMapsDefault(ctx context.Context, trustPolicies *TrustPolicies, reference string, namespace string) (map[PKKey]keymanagementprovider.PublicKey, cosign.CheckOpts, TrustPolicy, error) {
 	// get the trust policy for the reference
 	trustPolicy, err := trustPolicies.GetScopedPolicy(reference)
```

Comment:
one question out of scope: `getKeysMapsDefault` is the only implementation of `getKeysMaps` for now, just wonder if `	trustPolicy, err := trustPolicies.GetScopedPolicy(reference)` should be in the scope of `getKeysMaps`. In the future, if we have another `getKeysMaps` implementation, would we need just a policy passed in or have different logics to fetch the policy?

Reply:
yeah good point. the trust policy fetch shoould be separated from the key fetching done on the trust policy after. I'll make the change.